recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
```

Comment:
Here is a [tarball]( https://sourceforge.net/projects/netpbm/files/super_stable/10.47.60/ ) that we could use. Is this recent enough for you?